packages/react/src/components/Tooltip/Tooltip.module.css

```
@@ -0,0 +1,7 @@
+.wrapper {
+  background: var(--fds-semantic-border-neutral-strong);
+  padding: var(--fds-spacing-1) var(--fds-spacing-2);
+  color: white;
+  border-radius: var(--fds-spacing-1);
```

We have separate tokens for border-radius. 
```suggestion
  border-radius: var(--fds-border_radius-medium);
```